site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,74 @@
+
+![Octant Electron System Tray](/img/posts/2021/02/19/electron-system-tray.png)
+
+The header also went through a few changes. Since we are not operating inside the browser anymore, we had to add the Forward/Back buttons to the header. Additionally, header responsiveness has been improved to make it more usable at lower resolutions. 
+
+A thing to note: In this release macOS needs an additional step when running Octant for the very first time. You will have to locate Octant in Finder, control-click the Octant icon, then choose Open. More details about running unsigned apps on Mac are available [here](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac). This will be resolved in future releases.
+
+<h3>Anatomy of the Navigation</h3>
+
+
+Based on user feedback, we decided to redesign the Octant navigation in order to accommodate the growing number of Octant plugins.  The left navigation is now separated into two vertical panels: a left panel with a list of all known modules and each module has its own panel to the right. Module panels contain all module specific navigation actions as well as module description at the bottom. The module description section right now displays just the description text, we anticipate opening that up in the future to allow inclusion of more interactive UI elements. This is a great news for Octant plugin authors, giving their plugin better visibility and discoverability. 
+
+The first three modules at the top (Applications, Namespace and Cluster) are built in the Octant core and always included. They are followed by a list of all installed plugin modules with Plugins and Preferences always last in the list.
+
+![Octant Electron Navigation](/img/posts/2021/02/19/electron-navigation.png)
+
+
+Navigation still has both collapsed and expanded state to support both novice and advanced users. The navigation state is configured in Preferences and persisted between sessions, making it easy to customize your Octant instance.
+
+One of the goals of a navigation redesign was to move some infrequently used actions to the Preferences. As a result, theme and navigation collapsed state moved from the navigation to the Preferences UI.
+
+
+![Octant Electron Preferences](/img/posts/2021/02/19/electron-preferences.png)
+
+
+Preferences are another new feature geared towards improved configuration and customization. The Preferences dialog is always accessible through a button at the bottom left corner and offers access to all app wide preferences. Preferences also contains a Developer panel where logging verbosity can be set. This is also where developers can configure the external proxy for easier development. Expect to see that list growing in the future.
```

Comment:
Remove `also` from `This is also where developers...` since also is used in the previous sentence.